README.md

```diff
@@ -459,12 +459,12 @@ QUERY or SEARCH verbs, if supported.
 gives access to a class of operations that can modify an existing resource.
 There are two levels: write permissions on an existing resource allows modifying
 (but not deleting) that resource.
-In a REST API context, this would include `POST` and `PATCH`. This also
+In a HTTP context, this would include `POST` and `PATCH`. This also
 includes the ability to perform SPARQL queries that perform updates, if those
 are supported.
 For creating or deleting a resource, write permissions on the container of which
 the resources is a member, are also needed.
-In a REST API context, this would include `PUT` and `DELETE`.
+In a HTTP context, this would include `PUT` and `DELETE`.
```

Comment:
`In a HTTP context`
-> `In an HTTP context`

Reply:
Absoltely disagree with "not on the resource
itself" -- to delete a file you MUST have Write on the file as well as Write on the container.